wmn-data.json

```diff
@@ -2297,6 +2307,16 @@
         "known" : ["cucumberz", "Yasline"],
         "cat" : "gaming"
        },
+       {
+        "name" : "Habtium",
+        "uri_check" : "https://habtium.es/{account}",
+        "e_code" : 200,
+        "e_string" : "</span>Miembro desde",

```

Comment:
This might be language specific based upon the browser's locale. I do not see this text in the response within my USA-bases English language browser but using curl I do. Can you try to find something that may be immune to that locale change please?